css/layout.css

```diff
@@ -111,4 +111,16 @@ li {
     font-size: 22px;
   }
 
+  .modal > .modal-window{
+  position: fixed;
+  top: 350px;
+  left: 700px;
```

Comment:
This is not the best way to center a modal, since these measurements won't always be the center of the viewport. You can use flexbox or other equivalent tools to get consistent centering